src/assets.zig

```diff
@@ -641,7 +641,7 @@ pub fn loadWorldAssets(assetFolder: []const u8, blockPalette: *Palette, itemPale
 		if(addon.kind == .directory) {
 			const path = std.fmt.allocPrintSentinel(main.stackAllocator.allocator, "assets/{s}/blocks/textures", .{addon.name}, 0) catch unreachable;
 			defer main.stackAllocator.free(path);
-			std.fs.cwd().access(path, .{}) catch continue;
+			if(!main.files.cwd().hasDir(path)) continue;
```

Comment:
This is odd, we are iterating over it and then again checking if it exists?
I think it should only be reachable by the iterator if it actually exists. So this seems redundant.

Reply:
Actually I think it may have been there to check if we have access rights or other conditions where we cannot open/watch it.

Reply:
maybe add a comment here (and in the equivalent place below), so we know why it's done.